src/graphics/gfx_credits.cpp

```diff
@@ -90,7 +90,7 @@ void DrawCredits()
             break; // Nothing also to draw
 
         // Printing lines of credits
-        SuperPrint(c.Text,
+        SuperPrint(GetS(c.Text),
```

Comment:
In a case like this, REALLY wants GetS() to use the const reference to avoid copyings of string as much as possible.